thewalrus/internal_modes/fock_density_matrices.py

```diff
@@ -17,6 +17,7 @@
 
 import numpy as np
 import numba
+from scipy.special import factorial as fac
```

Comment:
Can we just use the fact from .utils or is there something different?

Reply:
Now it is just `factorial` from `scipy.special`

Reply:
Yup, good point.